scripts/lib/mkcloud-libvirt.sh

```diff
@@ -103,6 +103,11 @@ function libvirt_prepare()
 
 function libvirt_setupadmin()
 {
-    ${mkcloud_lib_dir}/libvirt/admin-config $cloud $admin_node_memory $adminvcpus $emulator $admin_node_disk "$localreposdir_src" "$localreposdir_target" > /tmp/$cloud-admin.xml
+    nvramtemplate=""
```

Comment:
why do you need to explicitely pass down the nvram template? that sounds wrong to me, libvirt has autodetection code for this, the default should be fine.